# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxWindowsPlatform.h
# Commit: 791eeb52f034
# Full Hash: 791eeb52f0346565055cd9c9a881661b1ca688c5
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-01-24 09:30:00
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r 353ef4721bba -r 791eeb52f034 gfx/thebes/gfxWindowsPlatform.h
--- a/gfx/thebes/gfxWindowsPlatform.h	Mon Jan 23 22:13:16 2023 +0000
+++ b/gfx/thebes/gfxWindowsPlatform.h	Mon Jan 23 22:13:17 2023 +0000
@@ -200,7 +200,13 @@
 
  protected:
   bool AccelerateLayersByDefault() override { return true; }
+
   nsTArray<uint8_t> GetPlatformCMSOutputProfileData() override;
+
+ public:
+  static nsTArray<uint8_t> GetPlatformCMSOutputProfileData_Impl();
+
+ protected:
   void GetPlatformDisplayInfo(mozilla::widget::InfoObject& aObj) override;
 
   void ImportGPUDeviceData(const mozilla::gfx::GPUDeviceData& aData) override;